### PR TITLE
Pettingzoo env

### PR DESCRIPTION
**Pull request by weepingwillowben**
* Added pettingzoo env. You are required to pass in a pettingzoo env and a name
* Added additional supersuit wrapper to atari environment. Also fixed multiagent experiment unit test and made it faster.

### CODE REVIEW

File: all/environments/multiagent_pettingzoo.py [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+import numpy as np
+import torch
+import gym
+from all.core import MultiAgentState
+from ._multiagent_environment import MultiagentEnvironment
+
+
+class MultiagentPettingZooEnv(MultiagentEnvironment):
+    '''
+    A wrapper for generael PettingZoo environments (see: https://www.pettingzoo.ml/).
```
Comment (cpnota) [inline all/environments/multiagent_pettingzoo.py:11]: "generael" should be "general"

File: all/environments/multiagent_pettingzoo_test.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import unittest
+import torch
+from all.environments import MultiagentPettingZooEnv
+from pettingzoo.mpe import simple_world_comm_v2
+
+
+class MultiagentPettingZooEnvTest(unittest.TestCase):
```
Comment (cpnota) [inline all/environments/multiagent_pettingzoo_test.py:7]: These tests could be a little more thorough. Ideally, all of the public methods are tested.

**Review by cpnota — COMMENTED** [review]
Looks good! My only request would be to add a few more unit tests similar to the Multiagent Atari unit tests.

**Comment by weepingwillowben** [comment]
MultiAgentPettingZooEnv should probably just be called PettingZooEnv, as pettingzoo is only for multiagent anyways.

**Review by cpnota — APPROVED** [review]

**Review by cpnota — DISMISSED** [review]

**Review by cpnota — APPROVED** [review]

**Pull request merged**